.markdownlint.yml

```diff
@@ -0,0 +1,39 @@
+# Configuration for markdownlint
+# https://github.com/DavidAnson/markdownlint#configuration
+
+default: true
+
+# Disabled Rules
+# https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md
+
+# Breaking rules (higher priority to fix)
+
+# Example: https://www.terraform.io/docs/providers/aws/r/ssm_patch_baseline.html
+MD031: false
+# Example: https://www.terraform.io/docs/providers/aws/d/ebs_default_kms_key.html
```

Comment:
Won't these comments be out of date at some point in the future?

Reply:
They will be removed when the fixes for these are merged immediately after this is merged.